campaign_info.toml

```diff
@@ -17,13 +17,13 @@ use_of_funds_source = "MediaWiki:WMDE_Fundraising/UseOfFunds_2023_DE"
 # Banners of the campaign, key after "banners" can be anything
 [desktop.banners.ctrl]
 filename = "./banners/desktop/banner_ctrl.ts"
-pagename = "B23_WMDE_Desktop_DE_22_ctrl"
-tracking = "org-22-231218-ctrl"
+pagename = "B23_WMDE_Desktop_DE_24_ctrl"
+tracking = "org-22-231224-ctrl"
```

Comment:
The tracking needs to be updated to 24 eg: `org-24-231224-ctrl`

Reply:
Done.